src/server.ts

```diff
@@ -12,15 +13,40 @@ if (!app.get("prod")) {
 /**
  * Start Express server.
  */
-const server = app.listen(app.get("port"), () => {
+const server = app.listen(app.get("port"), async () => {
     console.log(
         "  App is running at http://localhost:%d in %s mode",
         app.get("port"),
         app.get("env")
     );
     console.log("  Press CTRL-C to stop\n");
+    await setStorage();
+    console.log('Storage now live')
```

Comment:
Maybe make the log message a bit more clear.

Reply:
Updated